### ISSUE TITLE
Feature Request: Add "json" as valid output for "info" command

### ISSUE THREAD

**Issue by manuel-koch** [jandelgado/rabtap#58]
To ease parsing of rabtap output, it would be nice if "json" would be a valid output format, using it like
`rabtap info --stats --format=json`

**Comment by jandelgado** [comment]
I hesitated to implement this feature, because the REST API of RabbitMQ, which rabtap uses, does already provide this information: https://rawcdn.githack.com/rabbitmq/rabbitmq-server/v3.9.1/deps/rabbitmq_management/priv/www/api/index.html

rabtap offers an aggregated view combining different resources. Could you also use the REST API directly?

**Comment by manuel-koch** [comment]
Yes, I could use RabbitMQ REST API - just thought it would be convenient to use rabtap for that too.
